### PR TITLE
Correction of tips (content, trigger, ...)

### PR DESCRIPTION
**Pull request by qsm-odoo**
PAD : https://pad.odoo.com/p/r.JokowwPZJRXyfeZp
TASK : https://www.odoo.com/web#id=11037&view_type=form&model=project.task&menu_id=3942&action=327

### CODE REVIEW

File: addons/crm/crm_tip_data.xml [hunk 1]
```diff
@@ -14,38 +14,37 @@
...
-            <field name="description">This where you navigate through your process. Click here to go back to where you came from.</field>
+            <field name="title">Update your progress</field>
+            <field name="description">Here you can also change the status of the opportunity.</field>
             <field name="model">crm.lead</field>
             <field name="type">opportunity</field>
             <field name="mode">form</field>
             <field name="trigger_selector">.oe_form_buttons_view:visible,ol.breadcrumb a</field>
-            <field name="highlight_selector">ol.breadcrumb:visible a:last</field>
-            <field name="placement">bottom</field>
+            <field name="highlight_selector">.oe_form_field_status</field>
```
Comment (tivisse) [inline addons/crm/crm_tip_data.xml]: Don't break the flux please. Here the intention to explain the breadcrumb and to help the user during the process. 

If the user changes the Opportunity's stage, after we will propose him to drag and drop the task to change the stage => Redundancy

File: addons/hr/hr_tip_data.xml [hunk 2]
```diff
@@ -2,32 +2,31 @@
...
-            <field name="title"></field>
-            <field name="description">This is where you navigate through your process. Click here to go back to where you came from.</field>
+            <field name="title">Great</field>
+            <field name="description">Here is your applicant information. You can plan and start an interview. If it is positif, don't forget to create your new employee !</field>
             <field name="model">hr.applicant</field>
             <field name="mode">form</field>
-            <field name="trigger_selector">ol.breadcrumb:visible</field>
-            <field name="highlight_selector">ol.breadcrumb a:nth-child(3)</field>
-            <field name="placement">bottom</field>
+            <field name="trigger_selector">.oe_form_buttons_view:visible,div.oe_right.oe_button_box > button</field>
```
Comment (tivisse) [inline addons/hr/hr_tip_data.xml]: Same comment than above

File: addons/hr/hr_tip_data.xml [hunk 3]
```diff
@@ -2,32 +2,31 @@
...
+            <field name="highlight_selector">.oe_form_sheet:visible</field>
+            <field name="placement">auto top</field>
         </record>
 
         <record model="web.tip" id="hr_tip_3">
-            <field name="title"></field>
-            <field name="description">Congratulations! Your employee has been created. In this form, you can modify his personal information or even add a picture.</field>
+            <field name="title">Your employee</field>
```
Comment (tivisse) [inline addons/hr/hr_tip_data.xml]: Congratulations ?

File: addons/hr/hr_tip_data.xml [hunk 4]
```diff
@@ -2,32 +2,31 @@
...
+            <field name="description">Here is the employee information. Modify his personal information or even add a picture.</field>
```
Comment (tivisse) [inline addons/hr/hr_tip_data.xml]: Can't we check the action of the employee creation from the application form, in order to trigger the tip if and only if we're coming from that action ?

Here, the tip is triggered, even if I come from the Leaves menu, or the expense menu...

File: addons/web_tip/static/src/js/tip.js [hunk 5]
```diff
@@ -30,38 +25,20 @@
...
-            var self = this;
-            var action_id = action.id;
-            var model = action.res_model;
-        },
-
         on_view: function(view) {
             var self = this;
             var fields_view = view.fields_view;
-            var action_id = view.ViewManager.action ? view.ViewManager.action.id : null;
+            if(view.ViewManager.action)
```
Comment (tde-banana-odoo) [inline addons/web_tip/static/src/js/tip.js]: Please lint: spaces, use {}, ...

**Pull request closed without merging**